whatlies/embeddingset.py

```diff
@@ -679,6 +679,119 @@ def plot(
         )
         return self
 
+    def plot_3d(
+        self,
+        x_axis: Union[int, str, Embedding] = 0,
+        y_axis: Union[int, str, Embedding] = 1,
+        z_axis: Union[int, str, Embedding] = 2,
+        color: str = None,
+        axes_metric: Optional[Union[str, Callable, Sequence]] = None,
```

Comment:
To make it consistent with `plot` and `plot_interactive` let's use `axis_metric` instead of `axes_metric`.
I used `axes_metric` in `plot_interactive_matrix` because there was a `axes` argument. Maybe I shouldn't have done that!